### ISSUE TITLE
Disable default double-click behaviour

### ISSUE THREAD

**Issue by itsamiga** [danielmoniz/tower-defense#52]
Currently, double clicking inside the display window (ie. placing multiple towers rapidly [before the overlapping towers change]) highlights text outside the game display and causes the placing tower range circle to disappear momentarily.

Double clicks should likely not be a major part of the game, but it does help for overall tidiness and polish to control this aspect of how user input is handled.

**Comment by itsamiga** [comment]
(Also disabling default right-click behaviour)

**Comment by danielmoniz** [comment]
Right-click behaviour removed! Still need to remove the double-click.
